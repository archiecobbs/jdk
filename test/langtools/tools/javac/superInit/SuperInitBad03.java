/*
 * @test /nodynamiccopyright/
 * @bug 8194743
 * @summary Test invalid reference to "this" prior to superclass construction
 *
 * @compile/fail/ref=SuperInitBad03.out -XDrawDiagnostics SuperInitBad03.java
 */
class SuperInitBad03 {

    SuperInitBad03() {
        SuperInitBad03.super.hashCode();
        super();
    }
}
