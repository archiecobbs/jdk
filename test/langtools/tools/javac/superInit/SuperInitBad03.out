SuperInitBad03.java:11:23: compiler.err.cant.ref.before.ctor.called: super
1 error
